command/helper/helper_test.go

```diff
@@ -1,10 +1,9 @@
-package flags
+package helper
```

Comment:
Can we move this test to network tests? It really shouldn't be a part of the command directory

Reply:
Moved 👍 

[35422d4](https://github.com/0xPolygon/polygon-edge/pull/410/commits/35422d40ddd9a7013229a094cc426359578a564c)